publisher/kafkapubsub/kafkapubsub.go

```diff
@@ -0,0 +1,47 @@
+package kafkapubsub
+
+import (
+	"context"
+
+	"gocloud.dev/pubsub"
+	"gocloud.dev/pubsub/kafkapubsub"
```

Comment:
Can we blank import this, to match the GCP driver implementation?

Reply:
This is why we are using kafkapubsub.OpenTopic(), we are aiming to hide the implementation usage of gocloud within the publisher and avoid the ENV var.

Reply:
I'm indifferent about this, so happy as is